### PR TITLE
Scale canvas on window resize

### PR DESCRIPTION
**Pull request by JithinKS97**
Added automatic scaling of slide and its contents based on window size

### CODE REVIEW

File: client/src/components/top-bar/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default (props: any) => {
   return (
     <>
       <style>{getStyle(theme)}</style>
-      <Menu selectable={false} theme="dark" mode="horizontal">
+      <Menu className="menu" selectable={false} theme="dark" mode="horizontal">
```
Comment (anandsimmy) [inline client/src/components/top-bar/index.tsx:10]: @JithinKS97 shouldn't theme be dynamic?

Comment (JithinKS97) [reply client/src/components/top-bar/index.tsx:10]: Are you refering to the className that I added? I added that for providing a drop shadow.

Comment (anandsimmy) [reply client/src/components/top-bar/index.tsx:10]: @JithinKS97 no, theme='dark'..shouldn't that be theme={theme}

Comment (JithinKS97) [reply client/src/components/top-bar/index.tsx:10]: Theme="dark" is a prop provided by ant-design library. I think we can we can revisit this section later if we are planning to introduce another theme.

Comment (anandsimmy) [reply client/src/components/top-bar/index.tsx:10]: okay

**Pull request merged**